helm/chart-operator/templates/deployment.yaml

```diff
@@ -78,7 +78,11 @@ spec:
         command:
         - sh
         - -c
-        - until dig quay.io. {{ range (splitList "," .Values.externalDNSIP) }}@{{ . }} {{ end }}; do echo waiting for quay.io; sleep 2; done;
+        {{ if .Values.isManagementCluster }}
+        - until dig {{ .Values.registry.domain }}. {{ range (splitList "," .Values.externalDNSIP) }}@{{ . }} {{ end }}; do echo waiting for {{ .Values.registry.domain }}; sleep 2; done;
```

Comment:
`.Values.externalDNSIP` defaulted to `8.8.8.8` in China for me. I'm not sure this is right.

Reply:
I think it's better to ask @giantswarm/team-firecracker about this. Can we still use 8.8.8.8 in the China installations?